lib/ansible/modules/cron.py

```diff
@@ -33,8 +33,8 @@
   name:
     description:
       - Description of a crontab entry or, if env is set, the name of environment variable.
-      - Required if C(state=absent).
-      - Note that if name is not set and C(state=present), then a
+      - Required if I(state=absent).
+      - Note that if name is not set and I(state=present), then a
```

Comment:
I think this change makes finding the parameter values harder when they are italic instead of inline code. I think the [macros documentation](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation) should be updated so that parameters _and_ values are formatted as inline code for easier visual parsing.

Reply:
@acozine @samccann ^

Reply:
Hmm, since this has been the recommended way of writing things for a long time now, how about making `I(...)` more intelligent so that it formats `{var}={value}` in a different way, maybe using italics for `{var}` and teletype for `{value}`?

So something like: *state*=`present` or *state=*`present`

Reply:
That's much too jumbled. I still think the whole thing should be inline code since both the parameter name and the value will be used in code. Bold and italic are meant used to emphasize words for the benefit of the reader to know they have increased importance. Inline code indicates to the reader these are vales that will be used in code. Using italic to indicate values used in code sends confusing messages to the reader, IMO.

> Hmm, since this has been the recommended way of writing things for a long time now, 

It's never too late to change things for the better. 😄

Reply:
@samdoran thanks for the feedback. If we're going to use monospace for anything that will be used in code, we need to update [our own documentation about documenting modules](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation) and update modules across the board. I'll open an issue for that, and we'll discuss it in the [Docs WG](https://github.com/ansible/community/issues/521). I like your suggestion, but would like to see it implemented consistently, and meanwhile I'd like to get the rest of the fixes from this PR merged.